stub-idp/src/main/resources/assets/styles/application_auth.css

```diff
@@ -620,6 +620,16 @@ button:active, .button:active {
   transform: perspective(0) translateY(3px);
 }
 
+input.linkButton {
```

Comment:
to be really picky, with CSS classes we usually use `hyphen-delimited` naming convention